src/garage/tf/baselines/gaussian_cnn_baseline.py

```diff
@@ -27,6 +28,15 @@ def __init__(
             regressor_args=None,
             name='GaussianCNNBaseline',
     ):
+        assert isinstance(env_spec.observation_space, akro.Box) and \
```

Comment:
an assert is not the right way to go here. that will crash the program. asserts are only for testing conditions which absolutely should not be logically possible unless something terrible has happened, not for validating user input.

Here you just want a simple `if` and raise a ValueError